templates/home.html

```diff
@@ -16,7 +16,7 @@ <h1>Are we <em>web</em> yet?</h1>
   
   <p>
     Rust has mature and production ready frameworks in <a href="/topics/frameworks/#pkg-actix-web">Actix Web</a> and
-    <a href="/topics/frameworks/#pkg-rocket">Rocket</a>, and newer ones like <a href="/topics/frameworks/#pkg-warp">Warp</a>
+    <a href="/topics/frameworks/#pkg-axum">Axum</a>, and newer ones like <a href="/topics/frameworks/#pkg-warp">Warp</a>
```

Comment:
IMO, Axum is newer than Warp and Tide, so it would be better to change the sentence `newer ones like`.

Reply:
I agree. How about "innovative ones like"? I think that would be a great description for Warp, indicating that its API is quite different than one would be familiar with coming from other ecosystems. I'm not sure how well it fits Tide though, its API looks pretty straight forward and almost identical to Axum.

Reply:
done :+1: